source/includes/audit-log/_introduction.md

```diff
@@ -42,14 +42,14 @@ curl -X POST 'https://harvest.greenhouse.io/auth/jwt_access_token'
 <b>Note</b>: Use HTTPS for all requests. Requests made over HTTP will return an <code>HTTP 401</code> response.
 </aside>
 
-## Throttling
+## Rate Limiting
```

Comment:
Super small change, our branding is to only upper case the first word in titles. So lower case the second word 
```suggestion
## Rate limiting
```